packages/core/src/components/carousel/carousel-item/carousel-item.tsx

```diff
@@ -0,0 +1,29 @@
+import { Component, Element, Host, Prop, h } from '@stencil/core'
+@Component({
+  tag: 'atom-carousel-item',
+  shadow: false,
+})
+
+export class AtomCarouselItem {
+  @Prop({ mutable: true })
+  lazy?: boolean
+
+  @Element() host!: HTMLElement
+
+  // If you need more info please read the ADR 0013
```

Comment:
(suggestion) maybe we could link it?

Reply:
@felipefialho i was in doubt about that because we had to add the full doc link to it, and i was afraid of pollute the code

Reply:
uhmmm maybe we don't need add nothing, just do hahaha

Reply:
sorry @felipefialho i did not get it 😅

Reply:
done [4a70aa3](https://github.com/juntossomosmais/atomium/pull/263/commits/4a70aa341865effe6c181b5d744df087d171bf0b)